src/libs/zuora.ts

```diff
@@ -133,10 +133,19 @@ Additional Comms                                                   # reserved fo
     FROM
       RatePlanCharge
     WHERE
-      Product.ProductType__c = 'Newspaper - National Delivery' 
-      and RatePlanCharge.name = '${dayOfTheWeekName}' 
-      and RatePlanCharge.effectiveStartDate <= '${date}'
-      and (RatePlanCharge.effectiveEndDate >= '${date}' or (Subscription.autoRenew = true and Subscription.status = 'Active'))
+      (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') AND
+      Product.Name = 'Newspaper - National Delivery' AND
+      RatePlanCharge.name = '${dayOfTheWeekName}' AND
+      RatePlanCharge.effectiveStartDate <= '${date}' AND
+      (
+          RatePlanCharge.effectiveEndDate > '${date}' OR
+          (
+              RatePlanCharge.EffectiveEndDate >= '${today}' AND
```

Comment:
Now we are trying to simulate the auto renew - because auto renew only happens in the hours after a subscription ends, but we are predicting up to 14 days in advance.
This line checks the subscription will end in future (hence it would't have been auto renewed yet)